### ISSUE TITLE
Reaper segfaults when adjusting volume cc during playback, but only when plugin window is closed

### ISSUE THREAD

**Issue by vbgfhfngfdtyreyt** [sfztools/sfizz#980]
OS is Debian 10, DAW is Reaper 6, using sfizz 1.0.0, VST3.

I have a few soundfonts loaded up from VPO (but any soundfont has the same problem).

If I try and adjust the volume CC with the gui while the song is playing back, Reaper segfaults when I close the plugin window and only when I close the plugin window.

Same thing happens when I adjust volume CC on my keyboard, passing that CC to Reaper. (segfaults when the plugin window is closed but only then)

Unfortunately, adjusting volume is a very common operation so it results in a lot of segfaults.

It happens less frequently if I close the plugin window when playback is stopped, but not 100% sure about that.

**Comment by jpcima** [comment]
Reproduced, and pasting the backtrace for reference.
~~On which OS is this occurring?~~

```
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007fc5ea5ab353 in non-virtual thunk to VSTGUI::CVSTGUITimer::fire() () at /home/jpc/documents/projects/sfizz/plugins/editor/external/vstgui4/vstgui/lib/platform/../vstguibase.h:395
395			ptr->remember ();
[Current thread is 1 (Thread 0x7fc61bd50740 (LWP 2829527))]
(gdb) bt
#0  0x00007fc5ea5ab353 in non-virtual thunk to VSTGUI::CVSTGUITimer::fire() () at /home/jpc/documents/projects/sfizz/plugins/editor/external/vstgui4/vstgui/lib/platform/../vstguibase.h:395
#1  0x0000000000905b39 in  ()
#2  0x00007fc61bad36d8 in  () at /opt/REAPER/libSwell.so
#3  0x0000000000415666 in  ()
#4  0x00007fc61bd87b25 in __libc_start_main () at /usr/lib/libc.so.6
#5  0x000000000041ae0a in _start ()
quit) 
```

**Comment by jpcima** [comment]
It's expected to be fixed by #986.
Would you mind to check with one of the "Current" builds? https://sfz.tools/sfizz/downloads

**Comment by vbgfhfngfdtyreyt** [comment]
This is interesting...I didn't realize there were deb packages built. For whatever reason, I can reproduce the issue with the 1.0.0 tarball that I installed right after the 1.0.0 release, but I can't reproduce with the 1.0.0 deb package I just tried now. For that matter, I can't reproduce with the 64 bit "current" deb either (sfizz_1632423275.867cf2b4-0_amd64.deb) So it looks like it's fixed!

Thank you!!
